src/extension.ts

```diff
@@ -679,7 +681,7 @@ export function activate(context: vscode.ExtensionContext) {
               context.subscriptions
             );
           }
-        } catch(err) {
+        } catch (err) {
```

Comment:
print the error please

Reply:
I have printed the error message in logger.debug in next line.